pkg/crc/cache/cache.go

```diff
@@ -23,6 +24,14 @@ func New(binarName string, archiveURL string, destDir string) *Cache {
 	return &Cache{binaryName: binarName, archiveURL: archiveURL, destDir: destDir}
 }
 
+func NewOcCache(destDir string) *Cache {
+	return New(constants.OcBinaryName, constants.GetOcUrl(), destDir)
+}
+
+func NewPodmanCache(destDir string) *Cache {
+	return New(constants.PodmanBinaryName, constants.GetPodmanUrl(), destDir)
+}
+
```

Comment:
I think Gerard objected to having this in the generic `Cache` package, this could be moved to one of the preflight files for now.

Reply:
I mind to have the values IN the cache package as that makes it non-generic.
if they exist and get set outside the cache they would make this fiel really generic and reusable without having to modify this file... and just from constansts or the preflight that uses it

Reply:
This does not seem to be addressed in the latest iteration? Is there nothing to address?